linera-views/src/backends/value_splitting.rs

```diff
@@ -202,6 +203,84 @@ where
         Ok(big_values)
     }
 
+    fn read_multi_values_bytes_iter(
+        &self,
+        keys: Vec<Vec<u8>>,
```

Comment:
I wonder if it would make sense (maybe not in this PR) to also take a stream as an input. (Or just a regular iterator? Needs discussion…) That way, e.g. we wouldn't even put together the `big_key` for the 10000th element if the caller cancels the stream after the 5000th.